source/config/configSpec.py

```diff
@@ -13,7 +13,7 @@
 #: provide an upgrade step (@see profileUpgradeSteps.py). An upgrade step does not need to be added when
 #: just adding a new element to (or removing from) the schema, only when old versions of the config 
 #: (conforming to old schema versions) will not work correctly with the new schema.
-latestSchemaVersion = 10
+latestSchemaVersion = 11

```

Comment:
I am a bit concerned by cluttering NVDA's codebase with a config upgrade code for a config spec that has never been part of a release.

Couldn't we avoid this upgrade code and just inform on the lists that alpha users may have to reset manually their config to default?

Or if you fear that we do not reach all alpha users, an alternate possibility avoiding config upgrade is to change the name of the key, e.g. "useWasapi" instead of "wasapi". This way people having a "wasapi" in their config will just have a useless key in their config but no config issue.

Reply:
>Or if you fear that we do not reach all alpha users, an alternate possibility avoiding config upgrade is to change the name of the key, e.g. "useWasapi" instead of "wasapi". This way people having a "wasapi" in their config will just have a useless key in their config but no config issue.

Thanks for the suggestion.
I think I'll opt for using "WASAPI" capitalized, rather than lowercase, to solve this problem